src/store/mutations.js

```diff
@@ -217,4 +217,20 @@ export default {
 	removeMessage(state, {accountId, folderId, id}) {
 		Vue.delete(state.messages, normalizedMessageId(accountId, folderId, id))
 	},
+	subscribeFolder(state, {accountId, folderId, isSubscribed}) {
+		console.info('modifyFolderSubscription')
```

Comment:
I assume this is merely for debugging. please remove it :)

Reply:
Ok.